app/main.ts

```diff
@@ -0,0 +1,6 @@
+import { createScene } from "./src/scene";
+
+const app = document.querySelector<HTMLDivElement>("#app");
```

Comment:
Sowohl `app` als auch `el` haben eine **id**.
In diesem Beispiel greifen wir über den `querySelector` zu und typisieren den generischen Teil mit `HTMLDivElement`.